public/app/plugins/datasource/graphite/datasource.ts

```diff
@@ -453,6 +453,16 @@ export class GraphiteDatasource {
     return this.getFuncDefs();
   }
 
+  variableFormat(formatValue: Function) {
+    return (value: any, format: any, variable: any) => {
+      if (_.isArray(value) && value.length === 1) {
```

Comment:
I deliberately chose to avoid special-casing any graphite versions. This is sensible enough to be supported on all of them. I tried 0.9 and 1.1.5 with success on both.